app/models/user.rb

```diff
@@ -7,4 +7,8 @@ class User < ApplicationRecord
 
   has_many :transactions, dependent: :destroy
   has_many :authorizations, dependent: :destroy
+
+  def user_balance
```

Comment:
@sinpai `#user_balance` is already in `User` model, no need in adding `user_`, just `#balance` 😉